ci/tasks/stop-canaries.yml

```diff
@@ -0,0 +1,28 @@
+---
+platform: linux
+image_resource:
+  type: registry-image
+  source:
+    repository: govukpay/concourse-runner
+params:
+  AWS_REGION: eu-west-1
+run:
+  path: /bin/bash
+  args:
+    - -ec
+    - |
+      for canary in notifcatns_sndbx_test pymntlnk_sandbox_test cancel_sandbox_test card_wpay_3ds2ex_test \
+      card_wpay_3ds2_test card_wpay_3ds_test card_wpay_test card_stripe_3ds_test card_stripe_test card_sandbox_test \
+      s_card_stripe_test s_card_stripe_3d_test s_card_wpay_test s_card_wpay_3ds_test s_card_wpay_3ds2_test \
+      s_wpay_3ds2ex_test s_card_sandbox_test s_cancel_sandbox_test s_paylnk_sandbox_test s_notifications_test; do
+
+        status=$(aws --region $AWS_REGION synthetics get-canary --name $canary | jq -r '.Canary.Status.State')
+        echo "status: $status"
+        if [ $status == "RUNNING" ]
```

Comment:
I think uou'll need to double quote this variable `"$status"` if you're using single brackets - that's why we're seeing the `unary operator expected` error.